### PR TITLE
Spike/security dependency updates

### PR DESCRIPTION
**Pull request by bogdanadrianmarc**
This PR addresses issue https://github.com/epimorphics/hmlr-linked-data/issues/127 by updating dependencies according to dependabot security alerts

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -226,8 +226,8 @@ GEM
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.3.0)
-      loofah (~> 2.3)
+    rails-html-sanitizer (1.4.4)
+      loofah (~> 2.19, >= 2.19.1)
```
Comment (joescottdave) [inline Gemfile.lock:230]: Is this the same as saying minor releases, so long as greater than 2.19.1?

Comment (bogdanadrianmarc) [reply Gemfile.lock:230]: It is, yes, essentially saying `2.19.1 <= ... < 3.0`

**Review by joescottdave — APPROVED** [review]
LGTM

**Review by jonrandahl — APPROVED** [review]
Nice work @bogdanadrianmarc!

LGTM! 👍

**Pull request merged**